benchmarks/automated/tests/utils/apache_bench.py

```diff
@@ -178,6 +181,26 @@ def generate_csv_output(self, requests, concurrency, connection=None):
 
         return artifacts
 
-    def generate_report(self, requests, concurrency, connection=None):
+    def push_benchmark_metrics(self, artifacts, connection=None):
+        curr_instance_type = connection.run(
+            f"curl http://169.254.169.254/latest/meta-data/instance-type", warn=True
+        ).stdout
+
+        artifacts["instance_type"] = curr_instance_type
+
+        dashboard_context = os.getenv("BENCHMARK_CONTEXT", "DevTest")
```

Comment:
where is this set? Why are we using an environment variable here

Reply:
This enivronment variable is set in the Codebuild jobs that run the benchmark and populate the metrics, currently run internally at AWS. However, when this variable is not set i.e a developer runs the script outside of a codebuild/github_actions environment. The dashboard_context is set to "DevTest"

Reply:
Can we add a comment, something like AWS internal env?

Reply:
Added